### PR TITLE
[Doc] Update woqlClient.py docstrings

### PR DESCRIPTION
**Pull request by michaelgrahamevans**
See #124

### CODE REVIEW

File: terminusdb_client/woqlclient/woqlClient.py [hunk 1]
```diff
@@ -164,29 +444,32 @@ def create_database(
...
+        accountid : str
+            Account identifier.
+        label : str, optional
+            Database name.
+        description : str, optional
+            Database description.
+        prefixes
+            TODO
+        include_schema : bool
+            TODO
```
Comment (michaelgrahamevans) [inline terminusdb_client/woqlclient/woqlClient.py:463]: I am not sure how do describe these parameters. If `prefixes` is set then it goes unused. Any suggestions?

Comment (kevinchekovfeeney) [reply terminusdb_client/woqlclient/woqlClient.py:463]: prefixes (dict - optional) dict containing:
            doc: (string) the IRI to use when doc: prefixes are expanded (defaults to terminusdb:///data)
            scm: (string) the IRI to use when scm: prefixes are expanded (defaults to terminusdb:///schema)

Comment (kevinchekovfeeney) [reply terminusdb_client/woqlclient/woqlClient.py:463]: Also, there's a small bug, it should be just "schema" not "include_schema" 

        schema (boolean - optional) if set to true, a main schema graph will be created, if not set, only a main instance graph will be created

Comment (michaelgrahamevans) [reply terminusdb_client/woqlclient/woqlClient.py:463]: Thanks!

**Comment by Cheukting** [comment]
Mind rebasing it form master and edit the PR to merge it to master?

File: terminusdb_client/woqlclient/woqlClient.py [hunk 2]
```diff
@@ -152,31 +441,39 @@ def create_database(
         label=None,
         description=None,
         prefixes=None,
-        include_schema=True,
+        schema=True,
```
Comment (michaelgrahamevans) [inline terminusdb_client/woqlclient/woqlClient.py:444]: Now that this PR is targeting `master`, should I set this back to `include_schema`? @kevinchekovfeeney

Comment (kevinchekovfeeney) [reply terminusdb_client/woqlclient/woqlClient.py:444]: Actually yes please do - sorry for the confusion - the include_schema flag can be mapped easily to the internal API schema flag - this is a brand new option in the latest server release so we're still working it in properly, but yes, include_schema is good.

Comment (michaelgrahamevans) [reply terminusdb_client/woqlclient/woqlClient.py]: Thanks, changes made :slightly_smiling_face:

**Pull request merged**